.coveragerc

```diff
@@ -4,7 +4,12 @@
 omit =
     */test*
     */__init__.py
```

Comment:
They should be testable through the handlers. Actually, the coverage of those files are >85%!